content/en/continuous_testing/settings/_index.md

```diff
@@ -31,21 +32,33 @@ By default, all your tests running in CI/CD pipelines run sequentially (one afte
 
 Parallel tests are tests that run simultaneously in your continuous integration and continuous delivery (CI/CD) pipelines. 
 
-{{< img src="continuous_testing/continuous_testing_parallelization.png" alt="Setting parallelization for Continuous Testing" style="width:100%;">}}
+{{< img src="continuous_testing/parallelization_explained.png" alt="A diagram that explains the benefits of parallelization vs. sequential test runs" style="width:100%;">}}
 
 This ensures you can:
 
 * Reduce pipeline duration and ship new features faster
 * Increase development confidence and speed of delivery
 * Have complete test coverage and reduce the volume of production-breaking bugs from reaching your codebase
 
+### Estimate parallelization
+
+Click **Estimate Parallelization** to see how many tests Datadog recommends running in parallel based on your Continuous Testing metrics. 
```

Comment:
Just a suggestion, I had some trouble finding the estimate parallelization option in some accounts, maybe including it in the first UI screenshot would be helpful: https://a.cl.ly/4guNAY2x